website/content/en/blog/clusterlink-intro/index.md

```diff
@@ -28,12 +28,12 @@ In order to facilitate cross cluster communications, you could use Kubernetes
  as [Istio][], [Skupper][] and [Submariner][].
  By and large, these solutions attempt to conjoin the multiple clusters, flattening the
  isolated networks into a single flat "mesh" shared between the connected clusters.
- The goal is, to the extent possible, extend the Kubernetes single cluster network
- abstraction to multicluster use cases.
+ The goal is, to the extent possible, to extend the Kubernetes single cluster network
+ abstraction into multicluster use cases.
 
 The creation of a shared mesh is not always desirable and may place additional
  constraints on administrators, developers and the workloads they manage. For example,
- it might make assumptions on objects, such as Services, being the "same" based
+ it might make assumptions that objects, such as services, are the "same" based
```

Comment:
There are lengthy discussions regarding use of capitalization of k8s objects (Pod, Service, Volume) in the Kubernetes documentation SIG. Their decision is to use lower case when referring to the concepts in general and PascalCase when referring to the API objects directly (which is what we do here with Service and below with Pods)